examples/pettingzoo/meltingpot_env.py

```diff
@@ -0,0 +1,149 @@
+""" PettingZoo interface to meltingpot environments"""
+
+import dm_env
+from meltingpot.python import substrate
+from ml_collections import config_dict
+from gym import spaces
+import numpy as np
+from functools import lru_cache
+
+from gym.utils import EzPickle
+from pettingzoo.utils import wrappers
+from pettingzoo.utils.conversions import parallel_to_aec_wrapper
+from pettingzoo.utils.env import ParallelEnv
+import tree
+import matplotlib.pyplot as plt
+
+PLAYER_STR_FORMAT = 'player_{index}'
+MAX_CYCLES = 1000
```

Comment:
It would be useful to have a comment explaining what this constant is. It seems to be the maximum number of steps of the environment? Meltingpot substrates sometimes have longer time horizons than 1000 steps. Why does it need to have a hard cutoff?

Reply:
Appendix B in the Meltingpot paper on Substrate details states that "Episodes last 1000 steps" unless otherwise mentioned, so I replicated it for the default value.

It seems that Allelopathic Harvest is the only exceptio to this with 2000 steps, which can easily be passed in through the `max_cycles` argument to the constructor (or even `inf` for no cutoff)

Reply:
OK, just be aware that on future releases, this is likely to change. :)